projects/find/find.py

```diff
@@ -0,0 +1,37 @@
+import os
+import argparse
+
+"""This method is created to find all files and directories, given a specific path. Optionally, your can use -name flag
+to search only the files that match to a specific name"""
+
+
+def find(path, name=None):
+    if not os.path.exists(path):
+        return "Path doesn't exist or wrong: please try again"
+    stack = []
+    stack.append(path)
```

Comment:
you should initialize stack with `path` directly, instead of creating it empty and then add